onfido/resource.py

```diff
@@ -18,6 +18,7 @@ def build_url(self, path):
     @property
     def _headers(self):
         return {
+            "User-Agent": f"onfido-python/0.5.0",
```

Comment:
Lets make this dynamic by adding this to the module scope?

```python
import pkg_resources
CURRENT_VERSION = pkg_resources.get_distribution("onfido_python").version
```

Reply:
Thanks! :)

Reply:
I'm not sure if that works for invoking this within the package you're building - tried onfido_python, onfido-python and onfido and it doesn't do the trick :)

Reply:
That’s because we’re doing something very strange with our tests. We have a pyproject.toml but we’re also using a requirements.txt? We should execute `poetry install` as part of the tests, then it will actually install this package and the snippet will work 🤔

Reply:
[this bit](https://github.com/onfido/onfido-python/blob/31a8fb4f8ce9c4ddd8b539c8f67952ca3158528b/.github/workflows/python-package.yml#L29) should do `pip install poetry && poetry install`

Reply:
Oh that's my fault: I just ran the GitHub Workflows automagic setup :sweat_smile: 

Sorry @olasitarska

Reply:
Yup, sorted all of that out now. _Poetry_ is *lifechanging*.